### DISCUSSION TITLE
What is the use of multiple static features ?

### DISCUSSION THREAD

**Discussion by ArnMrchl** [Nixtla/mlforecast#202] [Q&A] [answered]
Not sure to understand how static features are used by the model.
Following what is said in the [example](https://nixtla.github.io/mlforecast/forecast.html#dynamic-features) : 

> Whatever extra columns you have, like static_0 and static_1 here are considered to be static and are replicated when constructing the features for the next timestamp

So what is the sense of multiple static columns if they are repeated for the next timestamp? A single static feature should be enough?

Thanks for your help,
Arnaud

**Comment by jmoralez** [comment, marked as answer]
Hey @ArnMrchl, thanks for using mlforecast. An example could be that you're forecasting sales and you have both product_id and store_id, these are both static features but they can influence the prediction in different ways. Hope this helps, please let us know if you have further doubts.

**Reply by ArnMrchl** [reply]
Thanks Jose. Still a doubt. In that case one should have a different `unique_id` for each combination product_id x store_id.

**Reply by ArnMrchl** [reply]
I finally got it ! Thanks
